### ISSUE TITLE
feature request: turtle

### ISSUE THREAD

**Issue by rlzijdeman** [netwerk-digitaal-erfgoed/LDWizard#75]
Hi, given the intend of ldwizard being a tool for novice LOD-ers, turtle might be a better (more readable) format to download triples in. Perhaps that could be an 'arrow down' solution in the gui, allowing you to choose multiple LD formats.

**Comment by wouterbeek** [comment]
Good idea, but I'm not aware of implementations that allow Turtle to be emitted in a somewhat readable way (using the correct abbreviations, etc.)  Having such an implementation available, maybe others have seen it somewhere?, makes it more feasible to complete this feature.
